plenum/test/view_change_with_delays/helper.py

```diff
@@ -42,6 +42,12 @@ def check_last_prepared_certificate(nodes, num):
         assert n.master_replica.last_prepared_certificate_in_view() == num
 
 
+def check_last_prepared_certificate_after_view_change_start(nodes, num):
+    # Check that last_prepared_certificate reaches some 3PC key on all nodes
+    for n in nodes:
+        assert n.master_replica.last_prepared_before_view_change == num
```

Comment:
Is equality here intended? I think this check can be made usable in more test scenarios by using `>=`

Reply:
In general yes. but in my (and I think the most of) test scenarios I needed strict equality.